utils/node_local_test.py

```diff
@@ -1134,6 +1134,44 @@ def test_open_unlinked(self):
             print('Failed to fstat() unlinked file')
         ofd.close()
 
+    @needs_dfuse
+    def test_symlink_broken(self):
+        """Check that broken symlinks work"""
+
+        src_link = os.path.join(self.dfuse.dir, 'source')
+
+        os.symlink('target', src_link)
+        entry = os.listdir(self.dfuse.dir)
+        print(entry)
+        assert len(entry) == 1
+        assert entry[0] == 'source'
+        os.lstat(src_link)
+
+        try:
+            os.stat(src_link)
+            assert False
+        except FileNotFoundError:
+            pass
+
+    @needs_dfuse
+    def test_symlink_rel(self):
+        """Check that relative symlinks work"""
+
+        src_link = os.path.join(self.dfuse.dir, 'source')
+
+        os.symlink('../target', src_link)
+        entry = os.listdir(self.dfuse.dir)
+        print(entry)
+        assert len(entry) == 1
+        assert entry[0] == 'source'
+        os.lstat(src_link)
+
+        try:
+            os.stat(src_link)
+            assert False
+        except FileNotFoundError:
+            pass
```

Comment:
This is really also testing a "broken" symlink, right? (Maybe one that's pointing outside the dfuse mount?)
FWIW, lookup does support:
`symlink -> ../target`
but it doesn't support consecutive ".." like this:
`symlink -> ../../target`
and it doesn't support ".." as the leaf:
`symlink -> dir/..`

So if there IS a `../target` (inside the container), then that should work.

Reply:
Yes, this is testing a broken symlink, where the link points outside the container.  I hadn't realized that one level of .. would work in dfs, although given this is in / itself then it won't in this case anyway.